easybuild/easyconfigs/l/likwid/likwid-4.2.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'likwid'
+version = '4.2.0'
+
+homepage = 'http://code.google.com/p/likwid/'
+
+description = """
+ Likwid stands for Like I knew what I am doing. This project contributes easy
+ to use command line tools for Linux to support programmers in developing high
+ performance multi threaded programs.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://github.com/RRZE-HPC/likwid/archive/4.2.0.tar.gz
+source_urls = ['https://github.com/RRZE-HPC/likwid/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1cb68f895cfce83b2d08a67bc4a0f8628202b59f2ea5949ef12061f3e7b03e94']
+
+patches = ['likwid-%(version)s-config-mk.patch']
```

Comment:
@JackPerdue Please move this up above `checksums`, and also include a SHA256 checksum for the patch file (`--inject-checksums` will do this for you)